std/src/main/scala/algebra/std/list.scala

```diff
@@ -28,7 +35,7 @@ class ListMonoid[A] extends Monoid[List[A]] {
 
   override def combineN(x: List[A], n: Int): List[A] = {
     val buf = mutable.ListBuffer.empty[A]
-      (0 until n).foreach(_ => x.foreach(buf += _))
+    (0 until n).foreach(_ => x.foreach(buf += _))
```

Comment:
You could also allocate the closure outside of the first foreach. But, yeah, sucks that these are things we need to talk about.